### ISSUE TITLE
not working with https

### ISSUE THREAD

**Issue by ndr-brt** [Yegorich555/webpack-mock-server#15]
When the `devServer` is set with `https: true`, every request get a 404.
On the webpack console this error appears
```
WebpackMockServer. Server is not available on 127.0.0.1:8079 
```
(the server was on port 8080)

**Comment by Yegorich555** [comment]
Yep. Only http supports for now. I'll take a look soon and respond you ASAP

**Comment by Yegorich555** [comment]
Fixed 15405e9998267b60c0f49144831dd787176340fa.

You can check it with `npm update webpack-mock-server`
